app/assets/stylesheets/components/_timeline.scss

```diff
@@ -114,5 +111,12 @@
 
 .progress-actions-links {
   margin-top: 20px;
-  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+// Styling sorting links on index pages
```

Comment:
I am soliciting opinions for better places to put this in our list of css.  
This is doing work on the index pages for proposals.  It is here because this file is exclusively used in proposals, but it doesn't really belong here... or anywhere I could see.  It also felt a little odd to put it in its own file.